price-feeder/README.md

```diff
@@ -49,6 +49,21 @@ $ price-feeder /path/to/price_feeder_config.toml
 
 ## Configuration
 
+### `telemetry`
+
+A set of options for the application's telemetry, which is disabled by default. An in-memory sink is the default, but Prometheus is also supported.
+```shell
+$ price-feeder /path/to/price_feeder_config.toml
+```
```

Comment:
please add a link to cosmos-sdk telemetry: https://github.com/cosmos/cosmos-sdk/blob/main/docs/core/telemetry.md